src/components/HorizontalThermometer/VaccinationsThermometer/VaccinationsThermometer.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { range } from 'd3-array';
+import { scaleBand } from 'd3-scale';
+import { ParentSize } from '@vx/responsive';
+import { Label } from './VaccinationsThermometer.style';
+import { thermometerBarHeight } from '../RiskLevelThermometer/RiskLevelThermometer.style';
+import { COLOR_MAP } from 'common/colors';
+
+interface Section {
+  labelValue: string;
+  color: string;
+}
+
+const thermometerSections: Section[] = [
+  {
+    labelValue: '40%',
+    color: COLOR_MAP.VACCINATIONS_BLUE[0],
+  },
+  {
+    labelValue: '50%',
+    color: COLOR_MAP.VACCINATIONS_BLUE[1],
+  },
+  {
+    labelValue: '60%',
+    color: COLOR_MAP.VACCINATIONS_BLUE[2],
+  },
+  {
+    labelValue: '70%',
+    color: COLOR_MAP.VACCINATIONS_BLUE[3],
+  },
+  {
+    labelValue: '',
+    color: COLOR_MAP.VACCINATIONS_BLUE[4],
+  },
+];
+
+const VaccinationsThermometer: React.FC<{ height: number; width: number }> = ({
+  height,
+  width,
+}) => {
+  const indexList = range(thermometerSections.length);
+  const scaleRect = scaleBand<number>().domain(indexList).range([0, width]);
+  const bandWidth = scaleRect.bandwidth();
+
+  return (
+    <svg width={width} height={height}>
+      <defs>
+        <clipPath id="rounded-corners">
```

Comment:
Maybe add a generated id in case there is more than one thermometer in a page

```tsx
<clipPath id={`rounded-corners-${uniqueRandomId}`}>
// ...
```